althea_kernel_interface/src/exit_client_counter.rs

```diff
@@ -6,33 +6,52 @@ use regex::Regex;
 
 impl KernelInterface {
     pub fn init_exit_client_counters(&self) -> Result<(), Error> {
-        self.add_iptables_rule("iptables", &["-w", "-A", "OUTPUT", "-o", "wg_exit"])?;
-        self.add_iptables_rule("iptables", &["-w", "-A", "INPUT", "-i", "wg_exit"])?;
-
+        self.run_command("iptables", &["-N", "exit-client-counter"])?;
```

Comment:
I'd really like to see some documentation in this function, it's difficult to read.